composer.json

```
@@ -20,6 +23,9 @@
       "Web2PrintToolsBundle\\": "src/"
     }
   },
+  "suggest": {
+    "pimcore/admin-ui-classic-bundle": "^1.0"
```

```suggestion
    "pimcore/admin-ui-classic-bundle": "Required for Pimcore 11"
```

Normally you would want to add a description in the suggest section, not the version